### DISCUSSION TITLE
s/cloudflare-monitor-dashboard-by-graphql-python-api

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#48] [Announcements]
# s/cloudflare-monitor-dashboard-by-graphql-python-api

构建好用的Cloudflare监控大盘

https://blog.ops-coffee.cn/s/cloudflare-monitor-dashboard-by-graphql-python-api

**Comment by zzfly256** [comment]
这个真不错

**Reply by ops-coffee** [reply]
相当不错
